docs/source/technical_tutorials/access/accessing_external_data.ipynb

```diff
@@ -76,7 +76,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    export MAAP_PGT="your_edl_token_here"
</pre><p>This is not EDL token, it is PGT token. maap-py internally converts PGT to EDL token. So, change by "your_PGT_token"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>